build.cake

```diff
@@ -0,0 +1,695 @@
+///////////////////////////////////////////////////////////////////////////////
+
+var version = ParseAssemblyInfo(AssemblyInfoPath).AssemblyVersion;
+
+if (isRunningOnAppVeyor)
+{
+    if (isTagged)
+    {
+        // Use Tag Name as version
+        version = BuildSystem.AppVeyor.Environment.Repository.Tag.Name;
+    }
+    else
+    {
+        // Use AssemblyVersion with Build as version
+        version += "-build" + EnvironmentVariable("APPVEYOR_BUILD_NUMBER") + "-alpha";
+    }
+}
+
+///////////////////////////////////////////////////////////////////////////////
+// DIRECTORIES
+///////////////////////////////////////////////////////////////////////////////
+
+var artifactsDir = (DirectoryPath)Directory("./artifacts");
+var nugetRoot = artifactsDir.Combine("nuget");
+
+var dirSuffix = configuration;
+var dirSuffixSkia = (isPlatformAnyCPU ? "x86" : platform) + "/" + configuration;
+var dirSuffixIOS = "iPhone" + "/" + configuration;
+
+var buildDirs = 
+    GetDirectories("./src/**/bin/" + dirSuffix) + 
+    GetDirectories("./src/**/obj/" + dirSuffix) + 
+    GetDirectories("./src/Markup/**/bin/" + dirSuffix) + 
+    GetDirectories("./src/Markup/**/obj/" + dirSuffix) + 
+    GetDirectories("./src/Android/**/bin/" + dirSuffix) + 
+    GetDirectories("./src/Android/**/obj/" + dirSuffix) + 
+    GetDirectories("./src/Gtk/**/bin/" + dirSuffix) + 
+    GetDirectories("./src/Gtk/**/obj/" + dirSuffix) + 
+    GetDirectories("./src/iOS/**/bin/" + dirSuffixIOS) + 
+    GetDirectories("./src/iOS/**/obj/" + dirSuffixIOS) + 
+    (DirectoryPath)Directory("./src/Skia/Avalonia.Skia.Android/bin/" + dirSuffix) + 
+    (DirectoryPath)Directory("./src/Skia/Avalonia.Skia.Android/obj/" + dirSuffix) + 
+    (DirectoryPath)Directory("./src/Skia/Avalonia.Skia.Android.TestApp/bin/" + dirSuffix) + 
+    (DirectoryPath)Directory("./src/Skia/Avalonia.Skia.Android.TestApp/obj/" + dirSuffix) + 
+    (DirectoryPath)Directory("./src/Skia/Avalonia.Skia.Desktop/bin/" + dirSuffixSkia) + 
+    (DirectoryPath)Directory("./src/Skia/Avalonia.Skia.Desktop/obj/" + dirSuffixSkia) + 
+    (DirectoryPath)Directory("./src/Skia/Avalonia.Skia.iOS/bin/" + dirSuffixIOS) + 
+    (DirectoryPath)Directory("./src/Skia/Avalonia.Skia.iOS/obj/" + dirSuffixIOS) + 
+    (DirectoryPath)Directory("./src/Skia/Avalonia.Skia.iOS.TestApp/bin/" + dirSuffixIOS) + 
+    (DirectoryPath)Directory("./src/Skia/Avalonia.Skia.iOS.TestApp/obj/" + dirSuffixIOS) + 
+    GetDirectories("./src/Windows/**/bin/" + dirSuffix) + 
+    GetDirectories("./src/Windows/**/obj/" + dirSuffix) + 
+    GetDirectories("./tests/**/bin/" + dirSuffix) + 
+    GetDirectories("./tests/**/obj/" + dirSuffix) + 
+    GetDirectories("./Samples/**/bin/" + dirSuffix) + 
+    GetDirectories("./Samples/**/obj/" + dirSuffix);
+
+///////////////////////////////////////////////////////////////////////////////
+// NUGET NUSPECS
+///////////////////////////////////////////////////////////////////////////////
+
+var SerilogVersion = "1.5.14";
```

Comment:
I think that should be extracted from packages.config from project (and trigger error if there is version mismatch in packages referenced from different projects)